cpp/oneapi/dal/graph/BUILD

```diff
@@ -13,7 +13,20 @@ dal_module(
     ]
 )
 
+_DATA_DEPS = [
+    "@onedal//cpp/oneapi/dal/graph:test_data",
+]
+
 dal_test_suite(
-    name = "tests",
-    tests = [],
+    name = "graph_tests",
+    framework = "catch2",
+    srcs = glob([
+        "service_functions_test.cpp",
+    ]),
+    dal_deps = [
+        ":graph",
+        "@onedal//cpp/oneapi/dal/io",
```

Comment:
Ideally, we should not depend on IO here. `load_graph` functionality should be tested separately.

Reply:
The only available opportunity to create graph is load_graph. Hence, we left this dependency at least for a now, until new constructors will be arrived.